src/ucp/core/ucp_rkey.c

```diff
@@ -221,23 +221,27 @@ ucs_status_t ucp_ep_rkey_unpack(ucp_ep_h ep, const void *rkey_buffer,
             ucs_assert(rkey_index < md_count);
 
             status = uct_rkey_unpack(p, &rkey->uct[rkey_index]);
-            if (status != UCS_OK) {
+            if (status != UCS_OK && status != UCS_ERR_UNREACHABLE) {
```

Comment:
this error case can be added as the last 'else' clause for the 'if' in line 230:
```
if (status == UCS_OK) {
} else if (status == UCS_ERR_UNREACHABLE) {
} else {
  // error
}
```